### PR TITLE
[RA2] Vendor -> vendor

### LINKED ISSUE #2324: [RA2] Correcting the capitalization of cluster and vendor
Correcting the capitalization of the following:
- Kubernetes Cluster
- cluster
- vendor

### PR DESCRIPTION
**Pull request by EditKoselak**
Partially closes #2324

### CODE REVIEW

**Comment by EditKoselak** [comment]
cluster to be corrected when #2292 is closed

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The specifications defined in this chapter will be detailed with unique
 identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
 `ra2.ch.001` for the first requirement in the Kubernetes Node section.  These
 specifications will then be used as requirements input for the Kubernetes
-Reference Implementation and any Vendor or Community Implementations.
+Reference Implementation and any vendor or Community Implementations.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:30]: in this case "Community Implementations" should also be lower case.

**Review by rgstori — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The specifications defined in this chapter will be detailed with unique
 identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
 `ra2.ch.001` for the first requirement in the Kubernetes Node section.  These
 specifications will then be used as requirements input for the Kubernetes
-Reference Implementation and any Vendor or Community Implementations.
+Reference Implementation and any vendor or community implementations.
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:30]: Not sure about this one; I thought VI / CI were capitalisable just the same as RI?

**Review by pgoyal01 — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter04.md (hunk 2 above)
Comment (EditKoselak) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:30]: Thanks for the comment! Other mentions of "Vendor Implementation" seems to refer to the chapter Vendor Implementation within the documentation. In this case, however, I think it refers to actual vendor implementations and therefore it should not be capitalized.

**Review by rgstori — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**